tests/src/JIT/HardwareIntrinsics/X86/Avx/Add.Double.cs

```diff
@@ -28,32 +28,41 @@ private static void AddDouble()
                 // Validates basic functionality works, using Unsafe.Read
                 test.RunBasicScenario_UnsafeRead();
 
-                // Validates basic functionality works, using Load
-                test.RunBasicScenario_Load();
+                if (Avx.IsSupported)
```

Comment:
We explicitly test that the ISA used for the `Load` and `LoadAligned` instruction is supported. This is important for scenarios like `Sse3` through `Sse42` where they may be fully implemented themselves but are dependent on another ISA that is not (such as `Sse2`).